src/installer/tests/HostActivation.Tests/NativeHosting/Ijwhost.cs

```diff
@@ -17,19 +17,65 @@ public class Ijwhost : IClassFixture<Ijwhost.SharedTestState>
     {
         private readonly SharedTestState sharedState;
 
+        private string IjwLibraryRuntimeConfigPath { get; }
+
         public Ijwhost(SharedTestState sharedTestState)
         {
             sharedState = sharedTestState;
+            // Create a runtimeconfig.json for the C++/CLI test library.
+            // This cannot be shared because some tests delete it
```

Comment:
The test should also be disposable and delete this, so that other tests always start out in the same state.

Reply:
Actually, maybe it would be clearer to make the `ijw` test directory into a `TestApp`/`TestArtifact` (that gets disposed by `SharedTestState`)? Then, the two cases where the runtimeconfig gets deleted/moved can copy it and do make whatever modifications needed to the copy (which will be deleted when the original gets disposed).

Reply:
Ok. I made the test disposable. I'm not 100% clear what your 2nd comment wants me to do. Are you saying that we wouldn't use generated runtimeconfig.json but we would have it somewhere in github and we would copy it from there?

Reply:
I was suggesting leaving it generated as part of the `SharedTestState` (instead of generating/deleting for each test) and then having the test cases that need to delete/move the runtimeconfig to make a copy of the entire folder containing the files (ijw.dll, iwjhost, runtimeconfig).

Reply:
Ok. I refactored the tests to make copies of the test artifacts when necessary.